### DISCUSSION TITLE
Como alterar o separador de decimais? (PoNumberSeparator)

### DISCUSSION THREAD

**Discussion by francisco7comm** [po-ui/po-angular#1746] [Tire suas dúvidas] [answered]
Preciso que o número decimal seja exibido na tabela usando a vírgula como separador (ex.: 12,00), como usado no nosso idioma, em vez do ponto-final (ex. 12.00, como usado por padrão/língua inglesa etc)

Encontrei a interface PoNumberSeparator que pelo que entendi trata desta questão, porém não consigo usar ela e não encontrei na documentação como configurar. Estou no caminho certo ou equivocado em como tratar esse problema?

Att,

**Comment by anliben** [comment]
Bom Dia @francisco7comm, tudo bem?

Estarei analisando em breve retorno uma resposta mais assertiva!

Ator.

**Comment by anliben** [comment, marked as answer]
Boa Tarde @francisco7comm, tudo bem?

Segue documentação de como formatar decimal na table.

```
/**
   * Formato de exibição do valor da coluna:
   * - Formato para moeda (currency). Exemplos: 'BRL', 'USD'.
   * - Formato para data (date): aceita apenas os caracteres de dia(dd), mês(MM) e ano (yyyy ou yy),
   * caso não seja informado um formato o mesmo será 'dd/MM/yyyy'. Exemplos: 'dd/MM/yyyy', 'dd-MM-yy', 'mm/dd/yyyy'.
   * - Formato para horário (time): aceita apenas os caracteres de hora(HH), minutos(mm), segundos(ss) e
   *  milisegundos(f-ffffff), os milisegundos são opcionais, caso não seja informado um formato o mesmo será
   * 'HH:mm:ss'. Exemplos: 'HH:mm', 'HH:mm:ss.ffffff', 'HH:mm:ss.ff', 'mm:ss.fff'.
   * - Formato para números (number): aceita um valor seguindo o padrão [**DecimalPipe**](https://angular.io/api/common/DecimalPipe)
   *  para formatação, e caso não seja informado, o número será exibido na sua forma original. Exemplo:
   *
   *  +  Com o valor de entrada: `50` e a valor para formatação: `'1.2-5'` o resultado será: `50.00`
   */
format?: string;
```

Siga as diretivas do angular: https://angular.io/api/common/DecimalPipe

La está escrito assim {{3.6 | number: '1.0-0'}}
Porem ira colocar apenas o que esta dentro de aspas simples 1.0-0


Para o caso do número 12,00 ele ficaria assim no método `p-columns`

```
...
{ property: 'code', type: 'number', format: '1.2-5' },
...
```

Se esta solução fizer sentido para você me retorne, por favor!

Attr.

**Reply by francisco7comm** [reply]
Olá, obrigado pelo retorno @anliben 

A minha dúvida era como fazer com que na `po-table` eu exibisse o número formatado. Eu consegui lidar com o problema usando pipe em um `p-table-cell-template`. Vendo a mensagem original notei que não mencionei esse dado e peço desculpas.

Att,

**Reply by anliben** [reply]
Boa Noite, é este método enviado é justamente para a tabela!

Espero ter ajudado 

Ator.
